README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Build Status](https://wso2.org/jenkins/buildStatus/icon?job=products/micro-integrator)](https://wso2.org/jenkins/job/products/job/micro-integrator/)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[<img src="https://img.shields.io/badge/Slack-@wso2--ei-blue">](https://wso2-ei.slack.com/messages/microintegrator/)
```

Comment:
Is there any possibility to change the display name to just **wso2-mi** instead of **@wso2-ei**

Reply:
wso2-ei is the workspace name so we cannot change it. I have changed the badge text to '@wso2-ei/microintegrator'